agent.go

```diff
@@ -440,6 +443,29 @@ func (s *sandbox) initChannel() error {
 	return s.channel.setup()
 }
 
+func grpcTracer(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (resp interface{}, err error) {
+	message := req.(proto.Message)
+	agentLog.WithFields(logrus.Fields{
+		"request": info.FullMethod,
+		"req":     message.String()}).Debug("new request")
```

Comment:
I wonder if changing log format will make log-parser happy

```go
agentLog.WithFields(logrus.Fields{
	"request": info.FullMethod,
	"req":     "new request"}).Debug(message.String())
```

Reply:
I don't think so. The problem is that kernel prints one line to the console while agent is printing the logrus format log and they mixed up. It is not even related to this patch at all. The situation can happen at any time. The reason we only see it until now is because it introduces a lot of agent logs while there are not much agent logs at all even if debug is on before this PR.